plugins/woocommerce-blocks/assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -76,18 +84,34 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 		},
 	};
 
+	const { cartItemsCount: cartItemsCountFromApi, cartIsLoading } =
```

Comment:
I don't feel we should be using `useStoreCart` in the editor, maybe we could just show a number depending on what option the merchant has selected?

`never`: show nothing
`always`: 0
`greater_than_zero`:  2

Reply:
Thanks for the suggestion, I have updated it accordingly.